user/links/template.py

```diff
@@ -0,0 +1,18 @@
+pkgname = "links"
+pkgver = "2.30"
+pkgrel = 0
+build_style = "gnu_configure"
+# TODO: configure fails with 'conditional "am__fastdepCXX" was never defined.'???
+configure_gen = []
+hostmakedepends = ["pkgconf"]
+makedepends = [
+    "libjpeg-turbo-devel",
+    "libtiff-devel",
+    "openssl3-devel",
+    "zlib-ng-compat-devel",
+]
+pkgdesc = "Lynx-like text WWW browser"
+license = "GPL-2.0-or-later"
+url = "http://links.twibright.com"
+source = f"{url}/download/links-{pkgver}.tar.gz"
+sha256 = "7f0d54f4f7d1f094c25c9cbd657f98bc998311122563b1d757c9aeb1d3423b9e"
```

Comment:
i tested this for a little bit and it seems to be working fine, but do please let me know if it's not
```suggestion
sha256 = "7f0d54f4f7d1f094c25c9cbd657f98bc998311122563b1d757c9aeb1d3423b9e"
hardening = ["vis", "cfi"]
```

Reply:
No issues when testing the changes you've proposed.